minmax_stack.rb

```diff
@@ -4,37 +4,60 @@
 # "maximum" _and_ "minimum" operations.
 
 
+
 class MinMaxStack
+
+  class DimensionError < StandardError;end
+
+  attr_accessor :items
+
   def initialize
+    @list = LinkedList.new
```

Comment:
There's no reason to use a `LinkedList` here — you should use a `Stack` (which you've already implemented).